optimizer_wrapper.rb

```diff
@@ -796,7 +802,7 @@ def self.apply_zones(vrp)
       next if zone.vehicles.compact.empty?
 
       zone.vehicles.each{ |vehicle|
-        vehicle.skills.each{ |skillset| skillset << zone[:id] }
+        vehicle.skills.each{ |skillset| skillset << zone[:id].to_sym }
```

Comment:
we need a discordantproblem raise here for vehicles and services in case the zone[:id] already exists as a skill. and a consistency check at the vrp.create level to refuse it as discordantproblem similar to what we do for reserved_internal_skill clash.